Formula/i/ipfs.rb

```diff
@@ -27,17 +26,20 @@ class Ipfs < Formula
   depends_on "go" => :build
 
   def install
-    system "make", "build"
-    bin.install "cmd/ipfs/ipfs"
+    ldflags = %W[
+      -s -w
+      -X github.com/ipfs/kubo.CurrentCommit=#{tap.user}
```

Comment:
Normally there should be commit hash

Reply:
If the commit hash is mandatory, then we have to keep the tag revision maybe?